test/realtime_web/channels/realtime_channel/presence_handler_test.exs

```diff
@@ -114,14 +111,14 @@ defmodule RealtimeWeb.RealtimeChannel.PresenceHandlerTest do
 
       for _ <- 1..100, reduce: socket do
         socket ->
-          assert {:reply, :ok, socket} =
+          assert {:reply, :ok, _} =
```

Comment:
This is the issue! socket needed to be passed around to be `reduced` and kept the policies that were cached etc!

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️